src/main/scala/com/microsoft/hyperspace/index/IndexConfig.scala

```diff
@@ -129,20 +144,38 @@ object IndexConfig {
     }
 
     /**
-     * Updates included columns for [[IndexConfig]].
+     * Updates included columns for [[IndexConfig.includedColumns]].
      *
      * Note: API signature supports passing one or more argument.
      *
-     * @param includedColumn included column for [[IndexConfig]].
-     * @param includedColumns included columns for [[IndexConfig]].
+     * @param includedColumn included column for [[IndexConfig.includedColumns]].
+     * @param includedColumns included columns for [[IndexConfig.includedColumns]].
      * @return an [[IndexConfig.Builder]] object with updated included columns.
      */
     def include(includedColumn: String, includedColumns: String*): Builder = {
```

Comment:
Just out of curiosity, why do we have these two args, not just `include(includedColumns: String*)`?
If we'd like to enforce non-emptiness, we could add assert here.

Reply:
This is a natural way to capture API's requirement in API's signature: "The API should be called with at least one argument" and it is used with builder pattern. As an example, you can check Spark's `Dataset` APIs that follow the builder pattern s.a. [`groupBy`](https://spark.apache.org/docs/latest/api/scala/org/apache/spark/sql/Dataset.html#groupBy(col1:String,cols:String*):org.apache.spark.sql.RelationalGroupedDataset) and [`agg`](https://spark.apache.org/docs/latest/api/scala/org/apache/spark/sql/Dataset.html#agg(expr:org.apache.spark.sql.Column,exprs:org.apache.spark.sql.Column*):org.apache.spark.sql.DataFrame)